editor/editor_about.h

```diff
@@ -54,21 +54,22 @@ class EditorAbout : public AcceptDialog {
 private:
 	void _license_tree_selected();
 	void _version_button_pressed();
-	ScrollContainer *_populate_list(const String &p_name, const List<String> &p_sections, const char *const *const p_src[], const int p_flag_single_column = 0);
+	void _item_with_website_selected(int id, ItemList *il);
+	void _item_list_resized(ItemList *p_il);
+	ScrollContainer *_populate_list(const String &p_name, const List<String> &p_sections, const char *const *const p_src[], const int p_single_column_flags = 0, const bool p_allow_website = false);
```

Comment:
```suggestion
	ScrollContainer *_populate_list(const String &p_name, const List<String> &p_sections, const char *const *const p_src[], int p_single_column_flags = 0, bool p_allow_website = false);
```
We don't use `const` with primitive parameters like int or bool.

Reply:
So I was right in thinking it SMELLED old.